### DISCUSSION TITLE
Is GPU hardware acceleration enabled on Linux?

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [FreeTubeApp/FreeTube#3616] [Q&A]
I ask because my Linux laptop gets hot (despite being fairy powerful) when playing videos through FreeTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @LinuxOnTheDesktop,
Yes, if your distro supports it then it is enabled in FT by default.

Related code: https://github.com/FreeTubeApp/FreeTube/blob/fc9d87d14e44e1649d7c674108da18bb8d471270/src/main/index.js#L167-L169

There is no way to disable it in FT yet. There is already an issue open to disable this https://github.com/FreeTubeApp/FreeTube/issues/2296

Please mark this as answered if this fully answered your question! :)

**Comment by LinuxOnTheDesktop** [comment]
I think that counts as a full answer - thank you! However, I an unsure whether Linux Mint (my distribution) supports hardware acceleration on Electron.
